_rules/__tests__/testcase-html-hint.js

```diff
@@ -28,6 +27,7 @@ const htmlHintRules = {
 	'space-tab-mixed-disabled': false,
 	'head-script-disabled': false,
 	'id-class-value': false,
+	'empty-tag-not-self-closed': false,
```

Comment:
Looks like the rule actually didn't exists in  the old version of HTML hint.
It is flagging all the `<br />` and `<img src="…" />` tags we have, and we do have a lot…